special_cases.md

```diff
@@ -0,0 +1,172 @@
+
```

Comment:
-  Intro to code quality
- You can use one or several type for lintering

- Lint with linter aggregator(overcommit, inquisition, 
- Lint with aggregator for Github mentions(pronto)
- Lint with run of each linter on CI